pkg/util/tls/tls.go

```diff
@@ -74,12 +72,27 @@ func (cfg *ClientConfig) GetTLSConfig() (*tls.Config, error) {
 	return config, nil
 }
 
+type ClientConfigWithEnabled struct {
+	ClientConfig `yaml:",inline"`
+	Enabled      bool `yaml:"tls_enabled"`
```

Comment:
Could you move the `Enabled` field before `ClientConfig`, so that in the auto-generated config the `tls_enabled` will be documented before the other options?

Reply:
I have done this in the latest master